nilearn/maskers/multi_nifti_masker.py

```diff
@@ -30,29 +24,32 @@ def _get_mask_strategy(strategy):
     elif strategy == 'epi':
         return masking.compute_multi_epi_mask
     elif strategy == 'whole-brain-template':
-        return partial(masking.compute_multi_brain_mask,
-                       mask_type='whole-brain')
+        return partial(
+            masking.compute_multi_brain_mask, mask_type='whole-brain'
+        )
     elif strategy == 'gm-template':
-        return partial(masking.compute_multi_brain_mask,
-                       mask_type='gm')
+        return partial(masking.compute_multi_brain_mask, mask_type='gm')
     elif strategy == 'wm-template':
-        return partial(masking.compute_multi_brain_mask,
-                       mask_type='wm')
+        return partial(masking.compute_multi_brain_mask, mask_type='wm')
     elif strategy == 'template':
-        warnings.warn("Masking strategy 'template' is deprecated."
-                      "Please use 'whole-brain-template' instead.")
-        return partial(masking.compute_multi_brain_mask,
-                       mask_type='whole-brain')
+        warnings.warn(
+            "Masking strategy 'template' is deprecated. "
+            "Please use 'whole-brain-template' instead."
+        )
+        return partial(
+            masking.compute_multi_brain_mask, mask_type='whole-brain'
+        )
     else:
-        raise ValueError("Unknown value of mask_strategy '%s'. "
-                         "Acceptable values are 'background', "
-                         "'epi', 'whole-brain-template', "
-                         "'gm-template', and "
-                         "'wm-template'." % strategy)
+        raise ValueError(
+            f"Unknown value of mask_strategy '{strategy}'. "
+            "Acceptable values are 'background', "
+            "'epi', 'whole-brain-template', "
+            "'gm-template', and 'wm-template'."
+        )
 
 
-@fill_doc
-class MultiNiftiMasker(NiftiMasker, CacheMixin):
+@_utils.fill_doc
```

Comment:
I was just consolidating imports, and there were several from `_utils`, so it seemed more efficient to just use `_utils.fill_doc` instead of import `fill_doc` separately.